src/main.ts

```diff
@@ -3,12 +3,28 @@ import * as runner from "./runner";
 import {messages, messagesFormatter} from './messages';
 import {logger, configureLogger} from "./logger";
 
+export interface BitbucketEnvs {
+    USER_EMAIL: string;
+    API_TOKEN: string;
+    BITBUCKET_REPO_SLUG: string;
+    BITBUCKET_COMMIT: string;
+    BITBUCKET_WORKSPACE: string;
+    BITBUCKET_CLONE_DIR: string;
+    BITBUCKET_API_URL: string;
+}
+
 export async function run(): Promise<void> {
     const args = minimist(process.argv.slice(2), {
-         boolean: ['debug'],
+         boolean: ['debug', 'help'],
          string: ['report', 'parasoftToolOrJavaRootPath'],
     });
 
+    // Show help messages if no parameters are set or '--help' parameter is set
+    if (args.length < 0 || args['help']) {
```

Comment:
Should this be `args.length <= 0` here if you want to show "help" when no parameters set?